swagger/swagger.json

```diff
@@ -435,37 +488,6 @@
         }
       },
       "description" : "A POST link related to a payment"
-    },
-    "selfAndNextLinks" : {
-      "type" : "object",
-      "properties" : {
-        "self" : {
-          "description" : "self",
-          "$ref" : "#/definitions/paymentLink"
-        },
-        "next_url" : {
-          "description" : "next_url",
-          "readOnly" : true,
-          "$ref" : "#/definitions/paymentLink"
-        },
-        "next_url_post" : {
-          "description" : "next_url_post",
-          "readOnly" : true,
-          "$ref" : "#/definitions/paymentPOSTLink"
-        }
-      },
-      "description" : "Resource self and next links of a Payment"
-    },
-    "selfLinks" : {
-      "type" : "object",
-      "properties" : {
-        "self" : {
-          "description" : "self",
-          "readOnly" : true,
-          "$ref" : "#/definitions/paymentLink"
-        }
-      },
-      "description" : "Resource self link of a Payment"
     }
   }
 }
```

Comment:
I checked and I have it setup in intelliJ. But given that this file is generated by the maven plugin and we don't hand-craft it this is what we get